Userland/Applications/PixelPaint/Tools/GradientTool.h

```diff
@@ -32,32 +32,52 @@ class GradientTool : public Tool {
     virtual StringView tool_name() const override { return "Gradient Tool"sv; }
 
 private:
+    enum class GradientMode {
+        Linear,
+        Radial,
+        __Count,
+    };
+
+    enum class IconStyle {
+        None,
+        ChangeWidthAndAngle,
```

Comment:
radial transversal and angle? 🤔

(optionally you could go for major and minor [semi]axes in an ellipsis…?)